pkg/config/metricsconfig.go

```diff
@@ -54,6 +55,21 @@ func (mcd *metricsConfig) GetMetricsRefreshInterval() time.Duration {
 	return mcd.metricsRefreshInterval
 }
 
+// CheckNamespace returns `true` if the namespace has to be considered
+func (mcd *metricsConfig) CheckNamespace(namespace string) bool {
+	// TODO(eddycharly): check we actually need `"-"`
+	if namespace == "" || namespace == "-" {
```

Comment:
What does `-` mean? The empty or default namespace?

Reply:
Looks like it's empty.